packages/legend-query-builder/src/stores/fetch-structure/QueryBuilderFetchStructureState.ts

```diff
@@ -64,6 +64,7 @@ export class QueryBuilderFetchStructureState {
       }
       case FETCH_STRUCTURE_IMPLEMENTATION.GRAPH_FETCH: {
         this.queryBuilderState.setIsCalendarEnabled(false);
+        this.queryBuilderState.setIsAdvancedModeEnabled(false);
```

Comment:
is there logic to turn this back when we toggle back to tds?

Reply:
the mode will be turned off when they toggle back to tds. When they are in graph fetch space advanced mode button would be disabled, when they switch back to tds the button would be editable again